core/src/OpenMPTarget/Kokkos_OpenMPTarget_Parallel.hpp

```diff
@@ -906,13 +910,15 @@ ThreadVectorRange(const Impl::OpenMPTargetExecTeamMember& thread, iType count) {
       iType, Impl::OpenMPTargetExecTeamMember>(thread, count);
 }
 
-template <typename iType>
+template <typename iType1, typename iType2>
 KOKKOS_INLINE_FUNCTION Impl::TeamVectorRangeBoundariesStruct<
     iType, Impl::OpenMPTargetExecTeamMember>
-ThreadVectorRange(const Impl::OpenMPTargetExecTeamMember& thread, iType begin,
-                  iType end) {
+ThreadVectorRange(const Impl::OpenMPTargetExecTeamMember& thread, iType1 begin,
+                  iType2 end) {
```

Comment:
Not consistent with the Cuda backend at least
https://github.com/kokkos/kokkos/blob/fe7e4e482d91252145e405c8f0e366b66179337b/core/src/Cuda/Kokkos_Cuda_Team.hpp#L664-L671

Reply:
Note that we might want to update this everywhere.